### PR TITLE
Pull changes from master into release-1.19

### PR DESCRIPTION
**Pull request by brandond**
#### Proposed Changes ####

Update release-1.19 with changes from master 

#### Types of Changes ####

* bugfix
* etcd
* kine
* ???
* profit

#### Verification ####

* Test recent fixes from master (etcd, etc)

#### Linked Issues ####

Related to #2398

### CODE REVIEW

**Review by erikwilson — APPROVED** [review]

**Review by Oats87 — APPROVED** [review]
LGTM

File: install.sh [hunk 1]
```diff
@@ -643,7 +680,9 @@ create_systemd_service_file() {
 [Unit]
 Description=Lightweight Kubernetes
 Documentation=https://k3s.io
+After=network-online.target
 Wants=network-online.target
+After=network-online.target
```
Comment (dweomer) [inline install.sh:685]: This line got duped some time ago, can we remove it now?

File: package/rpm/install.sh [hunk 2]
```diff
@@ -648,7 +648,9 @@ create_systemd_service_file() {
 [Unit]
 Description=Lightweight Kubernetes
 Documentation=https://k3s.io
+After=network-online.target
 Wants=network-online.target
+After=network-online.target
```
Comment (dweomer) [inline package/rpm/install.sh:653]: as above, duplicated line

**Review by dweomer — APPROVED** [review]

File: install.sh (hunk 1 above)
Comment (brandond) [reply install.sh:685]: I don’t want to fix anything in this PR, I want it to be just a straight pull of existing commits from master. I think there's a PR to fix this on master though, lets see if we can get that in.

**Pull request merged**